lib/Property.php

```diff
@@ -247,11 +247,11 @@ function serialize() {
         $str .= ':' . $this->getRawMimeDirValue();
 
         $out = '';
-        while (($len = strlen($str)) > 0) {
+        while (($len = \strlen($str)) > 0) {
             if ($len > 75) {
```

Comment:
@evert where does this 75byte limitation come from? is this defined in the spec?

Reply:
@staabm for vcard you can find it here: https://tools.ietf.org/html/rfc6350#section-3.2